cpp-inference/inference_scaling_imagenet.cpp

```diff
@@ -125,20 +130,21 @@ void run_mnist(const std::string& model_name,
   std::vector<double> run_model_times;
   std::vector<double> unpack_tensor_times;
 
-  int num_devices = get_num_devices();
-
   // create keys for models and scripts to split inferences accross mulitple
   // GPU devices.
   std::string model_key = "resnet_model_" + std::to_string(rank%num_devices);
   std::string script_key = "resnet_script_" + std::to_string(rank%num_devices);
+  if (use_multi) {
+    model_key = "resnet_model";
+    script_key = "resnet_script";
+  }
 
   MPI_Barrier(MPI_COMM_WORLD);
-  if(!rank)
+  if (!rank)
     std::cout<<"All ranks have Resnet image"<<std::endl;
 
   double loop_start = MPI_Wtime();
-  for (int i=0; i<100; i++) {
-
+  for (int i = 0; i < 101; i++) {
```

Comment:
any reason for changing this number? might be nice to have this be a env var or cli arg

Reply:
I bumped it by one to account for the warmup run we aren't including in the results